fili-core/src/main/java/com/yahoo/bard/webservice/web/ErrorMessageFormat.java

```diff
@@ -252,7 +252,10 @@
     ),
 
     TOO_MANY_BACKING_DATA_SOURCES("TableDataSource built with too many backing data sources: %s"),
-    TOO_FEW_BACKING_DATA_SOURCES("TableDataSource built with insufficient backing data sources: %s")
+    TOO_FEW_BACKING_DATA_SOURCES("TableDataSource built with insufficient backing data sources: %s"),
+    HAVING_CUSTOM_OPERATOR_WRONG_NUMBER_OF_PARAMETERS("Wrong number of parameters provided for %s operator. " +
+                                              "%s operator needs %d parameters but found %d"),
+    HAVING_CUSTOM_OPERATOR_IMPROPER_RANGE("Upper range cannot be less than the lower range for %s operator.")
     ;
```

Comment:
I think 'CUSTOM_OPERATOR' isn't informative here.  It really isn't a custom operator at this point, it's just an operator, the same as the others.